ui/package.json

```diff
@@ -246,10 +245,11 @@
     ]
   },
   "dependencies": {
-    "@hashicorp/design-system-components": "^1.8.0",
+    "@hashicorp/design-system-components": "https://github.com/hashicorp/design-system#head=app-sidenav-component&workspace=@hashicorp/design-system-components",
```

Comment:
The final step will be to update this to the latest version once it is published.